cg/meta/workflow/mip.py

```diff
@@ -193,37 +195,37 @@ def _open_bundle_file(self, relative_file_path: str) -> Any:
         full_file_path = Path(self.housekeeper_api.get_root_dir()).joinpath(relative_file_path)
         return yaml.safe_load(open(full_file_path))
 
-    def get_latest_metadata(self, family_id: str) -> dict:
-        """Get the latest trending data for a family."""
+    def get_latest_metadata(self, family_id: str) -> MipAnalysis:
+        """Get the latest trending data for a family"""
 
         mip_config_raw = self._get_latest_raw_file(family_id=family_id, tag=HkMipAnalysisTag.CONFIG)
-        qcmetrics_raw = self._get_latest_raw_file(
+        qc_metrics_raw = self._get_latest_raw_file(
             family_id=family_id, tag=HkMipAnalysisTag.QC_METRICS
         )
-        sampleinfo_raw = self._get_latest_raw_file(
+        sample_info_raw = self._get_latest_raw_file(
             family_id=family_id, tag=HkMipAnalysisTag.SAMPLE_INFO
         )
-        trending = dict()
-        if mip_config_raw and qcmetrics_raw and sampleinfo_raw:
+        mip_analysis = MipAnalysis
+        if mip_config_raw and qc_metrics_raw and sample_info_raw:
             try:
-                trending = parse_trending.parse_mip_analysis(
+                mip_analysis: MipAnalysis = parse_mip_analysis(
                     mip_config_raw=mip_config_raw,
-                    qcmetrics_raw=qcmetrics_raw,
-                    sampleinfo_raw=sampleinfo_raw,
+                    qc_metrics_raw=qc_metrics_raw,
+                    sample_info_raw=sample_info_raw,
                 )
-            except KeyError as error:
+            except ValidationError as error:
                 LOG.warning(
-                    "get_latest_metadata failed for '%s', missing key: %s",
+                    "get_latest_metadata failed for '%s', missing attribute: %s",
                     family_id,
                     error,
                 )
                 LOG.warning(
-                    "get_latest_metadata failed for '%s', missing key: %s",
+                    "get_latest_metadata failed for '%s', missing attribute: %s",
                     family_id,
                     error,
                 )
-                trending = {}
-        return trending
+                mip_analysis = MipAnalysis
```

Comment:
```suggestion
                mip_analysis = MipAnalysis()
```

Should it be like this?

Reply:
I do not know. Would that not actually call the class and lead to validation errors? I was trying to just return an "empty" object (failing in:
```
mip_analysis: MipAnalysis = self.analysis.get_latest_metadata(family_id=case_id)

        if not mip_analysis:
            return
```

Reply:
Returning MupAnalysis returns a class as of now. No its not empty, its a class object. Perhaps just do a return without anything in this case?

Reply:
How about this 2a88224?